pyth_observer/event.py

```diff
@@ -36,15 +34,7 @@ def __init__(self, check: Check, context: Context):
     async def send(self):
         # Publisher checks expect the key -> name mapping of publishers when
         # generating the error title/message.
-        if self.check.__class__.__bases__ == (PublisherCheck,):
-            text = cast(PublisherCheck, self.check).error_message(
-                self.context["publishers"]
-            )
-        elif self.check.__class__.__bases__ == (PriceFeedCheck,):
-            text = cast(PriceFeedCheck, self.check).error_message()
-        else:
-            raise RuntimeError("Invalid check")
-
+        text = self.check.error_message()
```

Comment:
Added publisher_name to publisher state and the code here is therefore simpler. 

Also with the previous logic if a new publisher was added this code would crash as it couldn't look up the name.